src/cloud/audiocom/audiocomcloudservice.cpp

```diff
@@ -0,0 +1,353 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "audiocomcloudservice.h"
+
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QBuffer>
+#include <QHttpMultiPart>
+
+#include "async/async.h"
+#include "containers.h"
+#include "types/translatablestring.h"
+
+#include "clouderrors.h"
+#include "network/networkerrors.h"
+
+#include "log.h"
+
+using namespace mu;
+using namespace mu::network;
+using namespace mu::framework;
+
+static const QString AUDIOCOM_CLOUD_TITLE("Audio.com");
+static const QString AUDIOCOM_CLOUD_URL("https://audio.com");
+static const QString AUDIOCOM_API_ROOT_URL("https://api.audio.com");
+static const QUrl AUDIOCOM_USER_INFO_API_URL(AUDIOCOM_API_ROOT_URL + "/me");
+
+static const QUrl AUDIOCOM_UPLOAD_AUDIO_API_URL(AUDIOCOM_API_ROOT_URL + "/audio");
+
+static QString audioMime(const QString& audioFormat)
+{
+    if (audioFormat == "mp3") {
+        return "audio/mpeg";
+    }
+
+    return "audio/x-wav";
+}
+
+AudioComCloudService::AudioComCloudService(QObject* parent)
+    : AbstractCloudService(parent)
+{
+}
+
+cloud::CloudInfo AudioComCloudService::cloudInfo() const
+{
+    return {
+        AUDIO_COM_CLOUD_CODE,
+        AUDIOCOM_CLOUD_TITLE,
+        AUDIOCOM_CLOUD_URL
+    };
+}
+
+cloud::AbstractCloudService::ServerConfig AudioComCloudService::serverConfig()
```

Comment:
I would suggest not using the term "config" here. It is not a configuration, but rather "information. Also, the method should be const

Reply:
Disagree, it is a configuration for servers